.travis.yml

```diff
@@ -0,0 +1,32 @@
+sudo: false
+dist: trusty
```

Comment:
do we want to use the newer Ubuntu version now? https://github.com/pelias/pelias/issues/575 tracks the progress of that task. we should do it soon but it doesn't have to be now. That said, the tests already work. If you want to keep it, you might as well also remove lines 11-13 and 15-20, since they're no longer required and will speed up the tests quite a bit.